.pre-commit-config.yaml

```diff
@@ -90,46 +91,45 @@ repos:
             # No direct imports from conftest
             conftest\ import|
             import\ conftest
-        types: [python]
-        files: ^pandas/tests/
+        types_or: [python]
     -   id: incorrect-code-directives
         name: Check for incorrect code block or IPython directives
         language: pygrep
         entry: (\.\. code-block ::|\.\. ipython ::)
-        files: \.(py|pyx|rst)$
+        types_or: [file, non-executable, cython, text, yaml, python, rst]
```

Comment:
This is the correct place to make the change :+1: 

The change here takes too much though, can you try
```
types_or: [python, cython, rst]
```
?

Likewise for the other local hooks of this kind

Reply:
Done!